thrifty-integration-tests/build.gradle

```diff
@@ -56,7 +56,17 @@ task compileTestThrift(type: Exec) {
     args('-jar', jarTask.archivePath.absolutePath, "--out=$projectDir/build/generated-src/thrifty", "$projectDir/ClientThriftTest.thrift")
 }
 
-tasks['compileTestJava'].dependsOn compileTestThrift
+
+task compileTestIncludeThrift(type: Exec) {
```

Comment:
I think it might be easier if, instead of using a custom Gradle task, this test were added to `LoaderTest.java` in `thrifty-schema`; adding Gradle task dependencies makes me nervous as they are picky and easy to break.